web/src/main/java/com/futurewei/alcor/web/entity/node/NodeInfo.java

```diff
@@ -60,6 +60,9 @@ public class NodeInfo implements Serializable {
     @QuerySqlField(index = true)
     private String ncm_id;
 
+    @JsonProperty("br-vxlan")
+    private String brVxlan;
```

Comment:
@DavidLiu506 I recommend calling the new field as "dataPathIp" in this PR, and then rename the "localIP" as "controlPathIp" in next PR. 

This way, we could make the IP usage very clear.  

P.S. we may not use vxlan for our data path, so don't title a name to a specific technique.